src/managers/WebhookManager.js

```diff
@@ -42,10 +42,19 @@ const splitBufferIntoChunks = (
 };
 
 const validateRequestType = (requestType: string): RequestType => {
-  if (!REQUEST_TYPES.includes(requestType)) {
+  const upperRequestType = requestType.toUpperCase();
```

Comment:
I think that if you cast this here it will fix everything
```
((requestType.toUpperCase(): any): RequestType)
```

Then you shouldn't need the changes later.